tutorials/monitoring-demo/walkthrough-setup.md

```diff
@@ -26,29 +25,15 @@ After unblocking the zip file, extract its contents into a folder. For the rest
 
 Open the `MonitoringDemo` folder and double-click on `run.bat`. This script will:
 
-1. Ask you to select a SQL Server to use as a transport. All of the components of the demo communicate by using a shared SQL Server database. You will be prompted to *use an existing SQL Server instance* or have the demo create it's *own dedicated SQL LocalDB instance*.
-2. You will be prompted to run the remaining steps with elevated privileges. This is needed so that processes can bind to their network ports and for SQL database creation.
-2. Create a LocalDB instance if required.
-2. Create the SQL database catalog with all of the necessary tables that the demo components will use.
-3. Update the configuration files for all of the components with the correct connection string details.
-4. Run ServiceControl components, each in their own window.
+1. You will be prompted to run the remaining steps with elevated privileges. This is needed so that processes can bind to their network ports.
+2. Update the configuration files for all of the components with the correct connection string details.
```

Comment:
Step two is not executed anymore. The LT folders are relative and hardcoded right?

Reply:
You are right. Good catch!